plugins/ingest/src/test/java/org/elasticsearch/plugin/ingest/transport/IngestActionFilterTests.java

```diff
@@ -225,4 +226,64 @@ public void run() {
         threadPool.shutdown();
     }
 
+    public void testApply_withBulkRequestWithFailure() throws Exception {
```

Comment:
can we not use underscores as part of method names? (unless we decided that we do and I missed it :) )